clang/test/Driver/msvc-link.cpp

```diff
@@ -0,0 +1,7 @@
+// REQUIRES: system-windows

```

Comment:
Add tests to existing msvc-link.c instead of a new file. Specify `--target=` to avoid reliance on the default target triple and avoid the need of REQUIRES.